syntax = "proto3";

package similarity;

option go_package = "https://github.com/tusxxx/KemSU-GoodLine;main";

service SimilarityService {
    rpc CalculateSimilarity(SimilarityRequest) returns (SimilarityResponse);
}

message SimilarityRequest {
    string image_base64_1 = 1;
    string image_base64_2 = 2;
}

message SimilarityResponse {
    float similarity = 1;
}